operator/pkg/ciliumendpointslice/manager.go

```diff
@@ -634,15 +634,16 @@ func (c *cesMgr) insertCESInWorkQueue(ces *cesTracker, baseDelay time.Duration)
 
 // Return the CES queue delay in seconds and reset cesInsert time.
 func (c *cesMgr) getCESQueueDelayInSeconds(cesName string) (diff float64) {
-	ces := c.desiredCESs.getCESTrackerOnly(cesName)
-	ces.backendMutex.Lock()
-	defer ces.backendMutex.Unlock()
-	timeSinceCESQueued := time.Since(ces.cesInsertedAt)
-	if !ces.cesInsertedAt.IsZero() {
-		var t time.Time
-		// Reset the cesInsertedAt value
-		ces.cesInsertedAt = t
-		diff = timeSinceCESQueued.Seconds()
+	if ces, ok := c.desiredCESs.getCESTracker(cesName); ok {
+		ces.backendMutex.Lock()
+		defer ces.backendMutex.Unlock()
+		timeSinceCESQueued := time.Since(ces.cesInsertedAt)
+		if !ces.cesInsertedAt.IsZero() {
+			var t time.Time
+			// Reset the cesInsertedAt value
+			ces.cesInsertedAt = t
+			diff = timeSinceCESQueued.Seconds()
+		}
 	}
```

Comment:
It would be great to get a log message here so that the fail is not completely silent.

Also I just realized that there are multiple usages of `getCESTrackerOnly` in this file, can you fix all of them and remove the function? Seems like using it is not safe and would lead to more panics in the future.